src/main/java/nextstep/courses/service/EnrollmentService.java

```diff
@@ -43,9 +40,8 @@ public void enrollSession(EnrollRequest request) {
 
         session.enrollStudent(request.getLoginUser(), request.getPayAmount());
 
-        sessionUsersRepository.deleteBySessionId(session.getId());
         List<SessionStudent> users = List.of(
-                new SessionStudent(session.getId(), loginUser.getId(), loginUser.getId())
+                new SessionStudent(session.getId(), loginUser.getId())
         );
         sessionUsersRepository.bulkSave(users);
```

Comment:
`List`로 만들어서 저장하는 대신에 `SessionStudent` 단일 객체로 저장해도 되지 않을까요?
```suggestion
        SessionStudent student = new SessionStudent(session.getId(), loginUser.getId());
        sessionUsersRepository.save(student);
```